.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+name: ci
+on: [push, pull_request]
+
+jobs:
+  build_and_deploy:
+    env:
+        # latest non-preview version at the time of writing this CI script
+        BASE_VERSION: 5.0.0.2012
+        DEFAULT_REF: refs/heads/5.0.0
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Download Nuget.exe
+      uses: NuGet/setup-nuget@v1.0.5
+      with:
+        nuget-version: 5.4.0
+    - name: setup-msbuild
+      uses: microsoft/setup-msbuild@v1
+    - name: Restore & Build
+      run: |
+        cd Xamarin.Forms.Platform.GTK
+        nuget restore Xamarin.Forms.Platform.GTK.csproj
+        msbuild Xamarin.Forms.Platform.GTK.csproj /property:Configuration=Release
+    - name: Package and upload
+      if: ${{ github.event_name == 'push' && github.ref == env.DEFAULT_REF }}
```

Comment:
you didn't get it, I said to remove the `push` filtering, and to only check the branch before calling nuget push